docs/src/tutorials/beginner_tutorials/train-from-scratch.rst

```diff
@@ -0,0 +1,160 @@
+Train a model from scratch
+##########################
+This tutorial explains how to train a model with ``metatrain`` from scratch and evaluate
+it. `This dataset`_ is used here as an example of the preferred dataset format. If you
+have your own dataset, you can simply replace the dataset file name with yours.
+
+.. _`This dataset`: https://raw.githubusercontent.com/metatensor/Workshop-spring-2025/refs/heads/main/training-custom-models/data/rmd17_ethanol_1000.xyz
```

Comment:
I would much rather use the ethanol example we already have in the repo. We don't know if this one is going to stay around for long